HaskellDates.hsproj/_HaskellDates.hsplay

```diff
@@ -0,0 +1,41 @@
+-- Haskell Playground 1.0
+now <- utctDay <$> getCurrentTime
+weekDay now
+
+let eventDay = addDays (-27) now
+weekDay eventDay
+
+let event = Event { date = eventDay, daysOfWeek = [3,5], week = -1 }
+
+let startDay = addDays (-15) now
+let endDay = addDays 15 now
+
+
+let allDays = takeWhile (<=endDay) $ (`addDays` startDay) <$> [0..]
+let validDays = filter (((flip elem) $ daysOfWeek event) . dayOfWeek) allDays
+
+-- filter (\x -> elem (dayOfWeek x) (daysOfWeek event)) allDays
+
+length validDays
+
+validDays
+
+
+
+
+
+
+
+
+
```

Comment:
I know it's crazy. If you pull down the repo and open it in Haskell for Mac you'll see. If you remove these lines, the output is cut off.